utils/00_unittest_baseline.sh

```diff
@@ -101,4 +101,3 @@ rm $FILE
 # -status
 
 # vim:ts=5:sw=5:expandtab
-
```

Comment:
Not sure I like those and the other empty line removals, especially in the end. That's just a matter of taste I guess

Reply:
I originally removed them because some files didn't have them and some files did, so I thought the intention was to not have them.

Reply:
ah, ok.